services/graphite/service.go

```diff
@@ -334,6 +336,14 @@ func (s *Service) handleLine(line string) {
 	// Parse it.
 	point, err := s.parser.Parse(line)
 	if err != nil {
+		switch err := err.(type) {
```

Comment:
I figured it would be better to have it as a switch so adding any additional cases in the future is just a single line. I also use the error value inside of the switch statement to check if the value is `NaN` or `Inf`, which are the two possibilities from this error. I'm currently leaving `Inf` as an error since graphite doesn't seem to handle it specially as part of their line protocol.